test/e2e/playwright.config.ts

```diff
@@ -70,5 +70,25 @@ export default defineConfig( {
 			name: 'iphone',
 			use: { ...devices[ 'iPhone 15 Pro' ] },
 		},
+		{
+			name: 'authentication',
```

Comment:
this project is needed to set some special stuff to try to avoid captchas etc. without polluting normal test runs